docs/docs/smart-contracts/PSP22/Extensions/capped.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 7
+title: PSP22 Capped
+---
+
+This example shows how you can implement a [PSP22](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/src/token/psp22) contract with a supply cap, analogue to [ERC20Capped](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/extensions/ERC20Capped.sol).
+
+## Step 1: Include dependencies
+
+Include `openbrush` as dependency in the cargo file or you can use [default `Cargo.toml`](/smart-contracts/overview#the-default-toml-of-your-project-with-openbrush) template.
```

Comment:
We've changed the template in the ocuemntation. COuld you check for example `PSP22Metadata` doc?